example/viam_example_app/lib/main.dart

```diff
@@ -74,6 +75,7 @@ class MyHomePage extends StatefulWidget {
 class _MyHomePageState extends State<MyHomePage> {
   bool _loggedIn = false;
   bool _loading = false;
+  late ResourceName? _cameraName;
```

Comment:
This should either be `late` or `optional`, no need for it to be both

Reply:
good catch, thank you!